docs/lakehouse-iceberg-catalog.md

```diff
@@ -264,16 +264,24 @@ You can pass [Iceberg table properties](https://iceberg.apache.org/docs/1.5.2/co
 
 The Gravitino server doesn't allow passing the following reserved fields.
 
-| Configuration item              | Description                                             |
-|---------------------------------|---------------------------------------------------------|
-| `comment`                       | The table comment.                                      |
-| `creator`                       | The table creator.                                      |
-| `location`                      | Iceberg location for table storage.                     |
-| `current-snapshot-id`           | The snapshot represents the current state of the table. |
-| `cherry-pick-snapshot-id`       | Selecting a specific snapshot in a merge operation.     |
-| `sort-order`                    | Selecting a specific snapshot in a merge operation.     |
-| `identifier-fields`             | The identifier fields for defining the table.           |
-| `write.distribution-mode`       | Defines distribution of write data                      |
+| Configuration item        | Description                                             | Default value | Required | Since Version |
+|---------------------------|---------------------------------------------------------|---------------|----------|---------------|
+| `comment`                 | The table comment.                                      | None          | No       | 0.2.0         |
+| `creator`                 | The table creator.                                      | None          | No       | 0.2.0         |
+| `current-snapshot-id`     | The snapshot represents the current state of the table. | None          | No       | 0.2.0         |
+| `cherry-pick-snapshot-id` | Selecting a specific snapshot in a merge operation.     | None          | No       | 0.2.0         |
+| `sort-order`              | Selecting a specific snapshot in a merge operation.     | None          | No       | 0.2.0         |
+| `identifier-fields`       | The identifier fields for defining the table.           | None          | No       | 0.2.0         |
+| `write.distribution-mode` | Defines distribution of write data                      | None          | No       | 0.2.0         |
```

Comment:
If the reason for the reservation could be added to the description, that would be even better.

Reply:
ok